spec/car_service_spec.rb

```diff
@@ -0,0 +1,72 @@
+require_relative '../service'
+
+
+describe Service do
+  context "During week time " do
+    before do
+      @monday = Time.new(2020, 6, 1, 10, 00, 00)
+      allow(Time).to receive(:now).and_return(@monday)
+
+      @service = Service.new
+    end
+
+    it 'When two clients come at the same time' do
+      expect(@service.add_reservation).to eq Time.now + 2 * 3600
```

Comment:
I would say that it's not very intuitive the way this looks. If I  see this method name `add_reservation`, I don't expect it to return the pickup time. Maybe it's best if this method is rather setting a `pickup_time` object or something and then you would get it from there. Or simply rename the method

Reply:
I did rename the method